configuration/config2/CfgStringLimits.cpp

```diff
@@ -0,0 +1,75 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include "CfgStringLimits.hpp"
+#include "EnvValue.hpp"
+
+std::string CfgStringLimits::getString() const
+{
+
+    std::string limitStr = "";
+
+    if (m_length != 0)
+    {
+        //limitStr += "string " + m_length + " charcters in length.";
```

Comment:
Just a note, If this were a final code review I'd ask that commented out code be removed.